src/external/pptm-factory.js

```diff
@@ -76,6 +76,12 @@ export function pptmFactory() : Object {
                 }
             };
         },
+        listenForButtonRender() {
+            window.paypalDDL = window.paypalDDL || [];
+            if (!window.paypalDDL.find(e => e.event === 'paypalButtonRender')) {
+                window.paypalDDL.push({ event: 'paypalButtonRender' });
```

Comment:
[Find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) isn't compatible with IE, and I'm not sure if webpack will take care of this and make it compatible. Maybe you can see if it will automatically transform it to `filter`, or maybe you can just use `filter` instead.

Reply:
@joshbeam Good point. I update to use filter method. Thanks!